js/src/elements/form-behavior.ts

```diff
@@ -625,10 +625,18 @@ function normalizeInputValue(target: EventTarget | null) {
 	}
 }
 
-function setBusySubmitButton(form: HTMLFormElement, isLoading: boolean) {
+function setBusySubmitButton(event: Event, form: HTMLFormElement, isLoading: boolean) {
+	const submitter =
```

Comment:
If there is no submitter, wouldn't it toggle every submit button in the form?

I wonder if this line can be simplified to `const submitter = (event as SubmitEvent).submitter;`

Then on line 635, it could be `if (submitter && submitter === element)`

If element has to be submit button element and submitter has to equal the element, then the submitter would also have to be a button element.

Reply:
This makes sense to me, I've updated accordingly